components/nimbus/src/lib.rs

```diff
@@ -19,28 +18,21 @@ pub use enrollment::{EnrolledFeature, EnrollmentStatus};
 pub use error::{NimbusError, Result};
 #[cfg(debug_assertions)]
 pub use evaluator::evaluate_enrollment;
-pub use matcher::AppContext;
 pub use schema::*;
 pub use targeting::NimbusTargetingHelper;
 
 cfg_if::cfg_if! {
     if #[cfg(feature = "stateful")] {
-        mod behavior;
-        mod client;
-        mod dbcache;
-        mod updating;
+        pub mod stateful;
 
-        pub mod nimbus_client;
-        pub mod persistence;
-
-        pub use crate::nimbus_client::*;
+        pub use stateful::nimbus_client::*;
+        pub use stateful::matcher::AppContext;
         pub use remote_settings::RemoteSettingsConfig;
```

Comment:
we don't need the `remote_settings::RemoteSettingsConfig` at all for stateless?

Reply:
No actually! That piece is handled in the Python code and the experiments are passed into the client object over FFI